src/backend/renderer/multigpu/mod.rs

```diff
@@ -839,15 +887,66 @@ where
         size: Size<i32, Physical>,
         dst_transform: Transform,
     ) -> Result<MultiFrame<'render, 'target, 'frame, R, T, Target>, Self::Error> {
-        if self.target.is_some() {
+        if let Some(mut target) = self.target.as_mut() {
             let buffer_size = size.to_logical(1).to_buffer(1, dst_transform);
-            let render_buffer = Offscreen::<Target>::create_buffer(self.render.renderer_mut(), buffer_size)
-                .map_err(Error::Render)?;
-            self.render
-                .renderer_mut()
-                .bind(render_buffer)
-                .map_err(Error::Render)?;
-        }
+            if target
```

Comment:
Is the size check enough? Are the formats/modifiers guaranteed to be the same over time?

Reply:
Dmabufs don't change format or modifiers, so by directly trying to export it we sidestep the problem, because:

- either this will succeed, then format and modifier is pinned.
- it doesn't succeed, which means we won't try again for the lifetime of the render_buffer and then it doesn't matter to us, if GL reallocates the buffer in the background. And we wouldn't want to try to export the buffer on every frame. Especially given that dmabufs are meant to be created right after creating and before use of the underlying object (as we found out from your `ExportDma` experiments).